### DISCUSSION TITLE
Strange errors on webdav sync

### DISCUSSION THREAD

**Discussion by promexio** [remotely-save/remotely-save#185] [General]
Hi:
I just try to migrate manually from Joplin to Obsidian.
To do so, I've installed this plugin to sync with my webdav server, already in use with a different folder and account with Joplin since some time.
Now, on Obsidian I get frequenly errors, see below.
Strangely I can bypass this by re-syncing until it works. And all of a sudden everything is synced. Sometimes one file is missing, so I restart syncing until it works also for this file. 
Have you any idea ? The same strange behaviour occurs on the iphone app!

SYSTEM INFO:
	Obsidian version: v0.15.9
	Installer version: v0.15.9
	Operating system: Darwin Kernel Version 21.6.0: Sat Jun 18 17:05:47 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T8101 21.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Remotely Save v0.3.25

 Errors:
1/ on several items:
plugin:remotely-save:216 remotely-save-1662284961126: abort sync, triggerSource=manual, error while getting_remote_files_list
plugin:remotely-save:216 Error: Request failed, status 400

2/ on several items:
remotely-save-1662284718590: abort sync, triggerSource=manual, error while syncing
eval @ plugin:remotely-save:216
plugin:remotely-save:216 AggregateError: 
    Error: Financial Assets/Ideas and Brainstorming/Renew energy.md: Request failed, status 400
        at eval (plugin:remotely-save:211:314339)
    at eval (plugin:remotely-save:211:314499)
    at Generator.next (<anonymous>)
    at a (plugin:remotely-save:177:280327)

**Comment by promexio** [comment]
Any ideas? Issue remains always the same.

**Comment by fyears** [comment]
no special chars. not limit from the plugin but from your remote service
